packages/marshal/test/prepare-test-env-ava.js

```diff
@@ -1,6 +1,8 @@
 // @ts-nocheck
+// eslint-disable-next-line import/no-extraneous-dependencies
 import '@endo/init/debug.js';
 
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { wrapTest } from '@endo/ses-ava';
```

Comment:
It is especially unclear to me how this dependency is extraneous. It deliberately eschew’s the ordering requirements, but is not extraneous by any measure.

Reply:
IIRC, the actual complaint is that the dependency should be in `dependencies` rather than `devDependencies`.

Reply:
That is an odd failure because our eslint-config is set up like:

```js
    "import/no-extraneous-dependencies": [
      "error",
      {
        "devDependencies": [
          "**/*.config.js",
          "**/*.config.*.js",
          "*test*/**/*.js",
          "demo*/**/*.js",
          "scripts/**/*.js"
        ]
      }
    ],
```

So anything under `test` should be running in the `dev` context and not require full-fledged `dependencies`.